pkg/event/target/elasticsearch.go

```diff
@@ -235,26 +235,22 @@ func createIndex(client *elastic.Client, args ElasticsearchArgs) error {
 
 // newClient - creates a new elastic client with args provided.
 func newClient(args ElasticsearchArgs) (*elastic.Client, error) {
-	client, clientErr := elastic.NewClient(elastic.SetURL(args.URL.String()), elastic.SetSniff(false), elastic.SetMaxRetries(10))
-	if clientErr != nil {
-		if !(errors.Cause(clientErr) == elastic.ErrNoClient) {
-			return nil, clientErr
-		}
-	} else {
-		if err := createIndex(client, args); err != nil {
-			return nil, err
+	client, err := elastic.NewClient(elastic.SetURL(args.URL.String()), elastic.SetMaxRetries(10))
```

Comment:
sniffer is set to True by default. There will be some sniffing happening in the background to discover nodes automatically in a running Elasticsearch instance. 

Do you think we need this?

Reply:
Yes it is recommended setting for clients @Praveenrajmani we shouldn't turn it off.